pkgs/top-level/all-packages.nix

```diff
@@ -10818,6 +10818,8 @@ with pkgs;
 
   netlify-cli = callPackage ../development/web/netlify-cli { };
 
+  nhost = callPackage ../development/web/nhost { };
```

Comment:
Please try to add this at the right place. The letter `h` comes after the letter `e`.